### ISSUE TITLE
Incorrect position of new maximized PDE

### ISSUE THREAD

**Issue by AmnonOwed** [processing/processing#1984]
The following occurs when you are working in a maximized PDE (the original PDE) and you either start a new sketch or open an existing one:
a) The new PDE is not maximized, but a window.
b) The new PDE window does have full screen dimensions.
c) The new PDE's position is randomly chosen somewhere on the screen.

So the result is a very large PDE window that is mostly off-screen. One can subsequently maximize it, however then the unconvenient PDE window dimensions and position remain. My current workaround is to switch the original PDE from maximized to window and then open a new sketch or existing one.

The expected and more convenient behavior would be to:
a) Start the new PDE maximized.
b) Set the new PDE dimensions to default or original PDE.
c) Set the new PDE position to default or original PDE.

In other words either use default behavior or copy the settings from the original PDE.

To replicate this issue, follow these steps:
1. Open the PDE.
2. Maximize it to full screen.
3. Press CTRL-N (New) a couple of times.

This will result in multiple new large PDE windows at random positions.

Tested with Processing 2.0.1. under Windows.
